pom.xml

```diff
@@ -428,17 +428,14 @@
                     <version>${dependency-check-maven.version}</version>
                     <configuration>
                         <!-- The following properties enable using a mirror for nist NVD data -->
-                        <cveUrl12Modified>${owasp.cveUrl12Modified}</cveUrl12Modified>
-                        <cveUrl20Modified>${owasp.cveUrl20Modified}</cveUrl20Modified>
-                        <cveUrl12Base>${owasp.cveUrl12Base}</cveUrl12Base>
-                        <cveUrl20Base>${owasp.cveUrl20Base}</cveUrl20Base>
+                        <cveUrlModified>${owasp.cveUrlModified}</cveUrlModified>
+                        <cveUrlBase>${owasp.cveUrlBase}</cveUrlBase>
                         <!-- End NVD mirror configuration -->
-                        <failBuildOnCVSS>7</failBuildOnCVSS>
+                        <!-- Commented out as we make a process for handling these -->
+                        <!--<failBuildOnCVSS>7</failBuildOnCVSS>-->
```

Comment:
So we are not failing for owasp on pr or nightly builds now? I thought we were just trying to stop it from failing PR builds.

Reply:
I defaulted to initially set the lowest barrier to re-entry and get it back in and not fail the builds primarily so that we could start getting reports. We could start failing nightly builds again, but I figured we could turn that back on pretty easily if needed.